SwiftGit2/Repository.swift

```diff
@@ -26,11 +26,11 @@ final public class Repository {
 		let result = git_repository_open(&pointer, URL.fileSystemRepresentation)
 		
 		if result != GIT_OK.value {
-			return failure(libGit2Error(result, libGit2PointOfFailure: "git_repository_open"))
+			return Result.failure(libGit2Error(result, libGit2PointOfFailure: "git_repository_open"))
 		}
 		
 		let repository = Repository(pointer)
-		return success(repository)
+		return Result.success(repository)
```

Comment:
You can also write `.success` here, if that would be more agreeable. (I don’t have strong feelings either way myself.)